roles/8-mgmt-tools/tasks/main.yml

```diff
@@ -23,6 +23,11 @@
     name: munin
   when: munin_install | bool
 
+- name: USAGE
+  Include_role:
```

Comment:
typo Include_role: s/b lower case 'i' use include_role: 

On a side note the use of '| bool' has changed again... see #2576